### ISSUE TITLE
Enhance 'create_view' method to include tables

### ISSUE THREAD

**Issue by earlmedina** [Esri/arcgis-python-api#512]
**Is your feature request related to a problem? Please describe.**
Currently, the 'create_view' method in arcgis/features/managers.py is written to only include layers in created views snippet included below:

```python
add_def = {
           "layers" : []
          }
if view_layers is None:
    for lyr in fs.layers:
        add_def['layers'].append(
        {
            "adminLayerInfo" : {
                "viewLayerDefinition" :
            {
                "sourceServiceName" : os.path.basename(os.path.dirname(fs.url)),
                "sourceLayerId" : lyr.manager.properties['id'],
                "sourceLayerFields" :  "*"
            }
            },
            "name" : lyr.manager.properties['name']
         })
```

**Describe the solution you'd like**
It would be simple to extend the method to include tables. This could be an optional parameter (e.g., let the user decide whether they want to include tables or allow them to choose which tables to include similar to the 'view_layers' parameter).

As an example, the above code could be updated to:

```python
add_def = {
           "layers" : [],
           "tables" : [] # Add this key
          }
if view_layers is None:
    for lyr in fs.layers:
        add_def['layers'].append(
        {
            "adminLayerInfo" : {
                "viewLayerDefinition" :
            {
                "sourceServiceName" : os.path.basename(os.path.dirname(fs.url)),
                "sourceLayerId" : lyr.manager.properties['id'],
                "sourceLayerFields" :  "*"
            }
            },
            "name" : lyr.manager.properties['name']
         })

    # Add the below loop
    for tbl in fs.tables:
        add_def['tables'].append(
        {
            "adminLayerInfo" : {
                "viewLayerDefinition" :
            {
                "sourceServiceName" : os.path.basename(os.path.dirname(fs.url)),
                "sourceLayerId" : tbl.manager.properties['id'],
                "sourceLayerFields" :  "*"
            }
            },
            "name" : tbl.manager.properties['name']
         })
```

**Additional context**
The above update is confirmed to work.

**Comment by earlmedina** [comment]
Please note this meant to be labeled as a Feature request/Enhancement. It doesn't look like I can update the label.

**Comment by CMPeng** [comment]
Fixed and added to v1.8.0 release, thank you @earlmedina for suggesting this.
